### DISCUSSION TITLE
Error:"The application will stop working at block height 193536."???

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2401] [Q&A]
While running 1.0.5 this popped up. Does this mean upgrading to 1.1.1 becomes mandatory after this block height?

**Comment by dorkmo** [comment]
yes upgrade to 1.1 series is mandatory for the implementation of transactions. i would recomend restarting computer before update, and after update just to make sure all the processes are closed
